addons/hr_holidays/tests/test_allocations.py

```diff
@@ -8,7 +8,7 @@
 from odoo.addons.hr_holidays.tests.common import TestHrHolidaysCommon
 
 
-@tagged('allocation')
+@tagged('allocation', 'jardin')
```

Comment:
What is this 'jardin' tag? :)

Reply:
oops, forgot to remove my test tag :eyes: